app/assets/stylesheets/views/_brexit_checker_results_page.scss

```diff
@@ -47,8 +47,12 @@
     border-bottom: none;
   }
 
-  .stay-updated__heading {
-    margin-top: govuk-spacing(6);
+  .stay-updated_description {
+    margin-bottom: govuk-spacing(4);
+    font-size: 20px;
```

Comment:
You could probably replace this `font-size` and media-query with a mixin instead: @include govuk-font(19). It depends how set we are on 20px on desktop and 16px below, as I think our standard font is normally 19px on desktop.

Reply:
Ah great, didn't know about `@include govuk-font(19)`.  Humm, I was running off the font-sizes in the prototype... but I would prefer to make it standard so will change that.

Does the govuk-font handle it's own responsive size change? What's out standard font size to drop it down to in the media query?

Reply:
`govuk-font` is responsive - 19px drops down to 16px. You can see the standard font-sizes here: https://govuk-design-system-preview.netlify.com/styles/typography/font-size/index.html I think whichever one you specify, it drops down to the next smallest on smaller devices.

Reply:
It works! Amended that, cheers these mixins are great.